src/Plugin/RetryPlugin.php

```diff
@@ -140,6 +141,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
             }
 
             $time = call_user_func($this->exceptionDelay, $request, $exception, $this->retryStorage[$chainIdentifier]);
+            \assert(is_int($time));
```

Comment:
Will this benefit the user?
If it is just to convince phpstan we can add comments.

Reply:
Better to add a `/**@var int */` or whatever. Agreed.

Reply:
this way the user gets a warning if the user func returns the wrong thing.

but i can also put the annotation.